### ISSUE TITLE
Sound of moving electrons continues when holding down keyboard nav control of balloon

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/balloons-and-static-electricity#547]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/721.
When moving the balloon with keyboard nav, the moving electron noise will play continuously, even if electrons aren't moving. This happens when holding the down, up, or right arrow keys against the wall.
**Steps to reproduce**
1. Make sure Show All Charges is selected
2. Use keyboard nav to rub balloon on sweater
3. Move balloon against wall
4. Use keyboard nav and hold down arrow continuously

**Visuals**

https://user-images.githubusercontent.com/41024075/138183057-5f7b1320-7a60-4065-8ee7-c2e39a9cbe3d.MOV

https://user-images.githubusercontent.com/41024075/138183058-af59c7c7-2ea2-4e62-b9e9-f5841531521d.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.2/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.2 2021-10-11 21:55:48 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36
Language: en-US
Window: 1280x649
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by Nancy-Salpepi** [comment]
This is fixed.
